common/src/main/resources/assets/minecraft_access/lang/en_us.json

```
@@ -59,6 +59,7 @@
   "minecraft_access.gui.access_menu_config_menu.button.fluid_detector_button": "Fluid Detector Configs",
   "minecraft_access.gui.camera_controls_config_menu.button.modified_rotating_angle": "Modified Rotation Angle",
   "minecraft_access.gui.camera_controls_config_menu.button.normal_rotating_angle": "Normal Rotation Angle",
+  "minecraft_access.gui.chat_screen.showing_message_range": "Showing messages %d threw %d",
```

Typo, can be fixed later via Weblate though.
```suggestion
  "minecraft_access.gui.chat_screen.showing_message_range": "Showing messages %d through %d",
```